src/frontend-samples/fire-decoration-sample/README.md

```diff
@@ -0,0 +1,30 @@
+# Particle Effect - Fire
+
+Copyright © Bentley Systems, Incorporated. All rights reserved.
+
+This sample show how to create a particle effect using fire as an example.
+
+## Purpose
+
+The purpose of this sample is to demonstrate the following:
+
+* Creating a decorator and rendering it in the active view.
+* Working with particle effects using the [ParticleCollectionBuilder](https://www.itwinjs.org/reference/imodeljs-frontend/rendering/particlecollectionbuilder) API.
+* The visualizing (NOT simulating) fire in an iModel.
+
+## Description
+
+This sample provides an example of a particle effect in world space (as apposed to view space).  The sample provides a tool to place and controls limitedly configure fire particle emitters.  Each emitter is initially configured based with preset parameters defined in [FireDecorationApp.ts](./FireDecorationApp.tsx).  Select which preset is use with the drop down menu.  Once you have deselected an emitter, there is no way to return to configuring it.
+
+For a basic decorator, the [Decorator](https://www.itwinjs.org/reference/imodeljs-frontend/views/decorator/) interface is implemented. The decorate method adds the [RenderGraphics](https://www.itwinjs.org/reference/imodeljs-frontend/rendering/rendergraphic/) to the given [DecorateContext](https://www.itwinjs.org/reference/imodeljs-frontend/rendering/decoratecontext). Once the decorator is created, pass it in the [ViewManager.addDecorator](https://www.itwinjs.org/reference/imodeljs-frontend/views/viewmanager/adddecorator/) method to have it rendered in all active views. This sample is specifically for working with particle effects. For examples of other decorators, see [Geometry Samples](https://www.itwinjs.org/sample-showcase/?group=Geometry+Samples&sample=simple-line-sample) and this sample's [HighlightEmitter](./FireDecorator.ts).
+
+Inside the decorate method, [ParticleCollectionBuilder.create](https://www.itwinjs.org/reference/imodeljs-frontend/rendering/particlecollectionbuilder/particlecollectionbuilder.create/) is called to create a builder for the particles.  Each particle is added to this builder and implements the [ParticleProps](https://www.itwinjs.org/reference/imodeljs-frontend/rendering/particleprops/?term=partic) interface.  For efficient memory usage, the textures used by the decorator is shared by all of them, and is disposed of when not used.
```

Comment:
/s/is shared/are shared
/s/is disposed of when not used/are disposed of when no longer in use